### PR TITLE
Keyboard controls

### LINKED ISSUE #2: No keyboard control
## Expected Behavior

For accessibility purposes you should be able to use the tab key to focus control to the component and then use the keyboard arrow controls and the enter key to select an object (or esc or tab to move away from the object). 
 
## Actual Behavior

Right now the only way to control the component is with a mouse, which would fail WCAG 2.0

## Steps to Reproduce the Problem

  1. Open the page
  2. Use the tab key to tab through the page
  3. The tab focus will not stop at the component but rather skips over the component and focuses on the next focusable item on the page

## Specifications

 - Platform: Microsoft Edge on Mac OS

### PR DESCRIPTION
**Pull request by Tintef**
Fixes #2 

## Proposed Changes

  - Add accesibility

### CODE REVIEW

File: src/components/Option.vue [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
       'vcs__option--disabled': disabled,
+      'vsc__option--focused': focused,
     }"
-    @mouseenter="$emit('openMenu', value, options)"
+    @mouseenter="$emit('openMenu', value, options, index, false)"
     @click="handleSelection"
+    @keydown.right="$emit('openMenu', value, options, index, true)"
+    @keydown.left="$emit('closeMenu')"
+    @keydown.down="$emit('nextOption')"
+    @keydown.up="$emit('prevOption')"
```
Comment (maurocen) [inline src/components/Option.vue:14]: Are these events emitted just when the `Option` is focused or at all times?

Does `@keydown.left` close all menus or just the last one? I think it should only close the last one.

**Review by maurocen — COMMENTED** [review]
I think this looks great. I just want to confirm it's possible to switch between keyboard/mouse without strange behaviors.

File: src/components/Option.vue (hunk 1 above)
Comment (Tintef) [reply src/components/Option.vue:14]: Hey, they are triggered only on the focused option. And yes, it only closes the last menu

**Review by maurocen — APPROVED** [review]
Looks good to me 🎉

**Pull request merged**